core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -1509,10 +1509,10 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     assert(Utils.isPushBasedShuffleEnabled(conf) === false)
     conf.set(SHUFFLE_SERVICE_ENABLED, true)
     conf.set(SparkLauncher.SPARK_MASTER, "yarn")
-    conf.set("spark.yarn.maxAttempts", "1")
+    conf.set("spark.yarn.maxAppAttempts", "1")
     conf.set(SERIALIZER, "org.apache.spark.serializer.KryoSerializer")
     assert(Utils.isPushBasedShuffleEnabled(conf) === true)
-    conf.set("spark.yarn.maxAttempts", "2")
+    conf.set("spark.yarn.maxAppAttempts", "2")
```

Comment:
While this test is no longer required (since we dont disable if multiple attempts are present - like we used to earlier), give it is a minor change, it is fine to include in this PR.